lib/ibmdb.js

```diff
@@ -399,6 +414,7 @@ IBMDB.prototype.buildInsertDefaultValues = function(model) {
  */
 IBMDB.prototype.updateOrCreate = IBMDB.prototype.save =
   function(model, data, options, callback) {
+    debug('IBMDB.prototype.updateOrCreate (enter) ', model, data, options);
```

Comment:
might want to list log the names of each item too -- `...model: %j, data: %j, options: %j, model, data, options)`